personal_data_labels.js

```diff
@@ -15,6 +15,11 @@ module.exports = Object.freeze({
   homephone: 'Home Phone Number (Land Line)',
   mobilephone: 'Mobile Phone Number',
   faxnumber: 'Fax Number',
+  twitter: 'Twitter',
```

Comment:
shouldn't these be `twitter handle` and `linkedin url` etc?

Reply:
yeah, updated